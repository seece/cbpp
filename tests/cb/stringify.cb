#define makro(_name) print "my name is " + #_name

makro(gustav)
