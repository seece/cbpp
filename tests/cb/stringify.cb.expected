// #define makro(_name) print "my name is " + #_name

print "my name is " + "gustav"
